tests/_runtime/test_executor.py

```diff
@@ -0,0 +1,67 @@
+from tests.conftest import ExecReqProvider, MockedKernel
```

Comment:
Would run tests over in tests/_ast/compiler.py (since no longer a executor change)

Try
```python
1
```

```python
1;
```

```python
expression = 1;
```

```python
1;2;3;4
```

```python
1;2;3;4;
```

```python
1; # Has a comment
```

```python
1 # Has a comment;
```

```python
mo.md("# splits on # are less than ideal") # Contains a #
```

```python
mo.md("# splits on # are less than ideal"); # Contains a #
```

and ensure expr is const None in the ";" cases